_includes/templates/compatibility-page.md

```diff
@@ -0,0 +1,210 @@
+
+{% assign tested = tool.rbf.tested. %}
+**Tested**: *{{tested.platforms}}* {% if tested.version
+!= "n/a" %} *version {{tested.version}}* {% endif %}
+
+**Tested on**: *{{tested.date}}*
+
+**What is Replace-by-Fee (RBF)?** An unconfirmed transaction can be replaced by another version of the
+same transaction that spends the same inputs.  Most full nodes support
+this if the earlier transaction enables [BIP125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) signaling and the
+replacement transaction increases the amount of fee paid.  In terms of
+block chain space used, this is the most efficient form of fee bumping.
+
+### Receiving support
+
+{:id="receive-notification"}
+{% assign rbf = tool.rbf.features. %}
+{% case rbf.receive.notification %}
+  {% when "true" %}{:.feature-yes}
+  - **Notification notes RBF**<br>
+    Notification of incoming transaction notes that the transaction is RBF signaling
+  {% when "false" %}{:.feature-bad}
+  - **Notification does not note RBF**<br>
+    Notification of incoming transaction does not note that the transaction is RBF signaling
+  {% when "na" %}{:.feature-neutral}
+  - **No notification**<br>
+    There are no incoming transaction notifications for this service
+  {% when "untested" %}{:.feature-neutral}
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
+  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
+{% endcase %}
+
+{:id="receive-list"}
+{% case rbf.receive.list %}
+  {% when "true" %}{:.feature-yes}
+  - **Received transaction labeled replaceable in list**<br>
+    Visually indicates that an incoming transaction has signaled RBF
+  {% when "false" %}{:.feature-bad}
+  - **Received transaction not labeled replaceable in list**<br>
+    Does not visually indicate that an incoming transaction has signaled RBF
+  {% when "na" %}{:.feature-neutral}
+  - **This services does not handle incoming transactions**<br>
+    This wallet doesn't support incoming transactions
+  {% when "untested" %}{:.feature-neutral}
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
+  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
+{% endcase %}
+
+{:id="receive-details"}
+{% case rbf.receive.details %}
+  {% when "true" %}{:.feature-yes}
+  - **Received transaction labeled replaceable in transaction details**<br>
+    Visually indicates that a received transaction has signaled RBF when viewing the transaction details
+  {% when "false" %}{:.feature-bad}
+  - **Received transaction not labeled replaceable in transaction details**<br>
+    Does not visually indicate that a received transaction has signaled RBF when viewing the transaction details
+  {% when "na" %}{:.feature-neutral}
+  - **Does not show transaction details**<br>
+    This service does not show transaction details natively. Usually this means the service links off to a block explorer for transaction details.
```

Comment:
I think the word "off" can be removed.  Edit: this same change should be made in other places where this same text is reused.